src/Containers/SlidingOutPanels.tsx

```diff
@@ -6,7 +6,7 @@ interface ImageProps {
     id: string;
     imageSource: string;
     alt: string;
-    imagePanelClick?(imageLink: string): void;
+    imagePanelClick?(imageLink: string): Function;
```

Comment:
Not sure if this is the right syntax btw?

imagePanelClick?: (imageLink: string) => Function;

Reply:
the other syntax works too, changed to the suggested syntax